cli/src/config/merge_tools.toml

```diff
@@ -71,10 +82,16 @@ merge-args = ["--wait", "--merge", "$left", "$right", "$base", "$output"]
 merge-tool-edits-conflict-markers = true
 # VS Code prefers Git-style conflict markers
 conflict-marker-style = "git"
+# `code --diff` does not support dir-diff
```

Comment:
Just question. Does `vscode` work with `diff-invocation-mode = file-by-file`? (It would be practically bad to spawn GUI for each file, though.)

Reply:
Good catch! It's not super-convenient, but it does work.

There's a weird trade-off: with the "--wait" argument, it's inconvenient. Without it, it's more convenient, but VS Code shows all files as deleted. I'll do with `--wait`, as that's the more predictable behavior.